### PR TITLE
Documentation for rest operation parameters

### PR DESCRIPTION
**Pull request by joostverhoog**
Rest operation parameters can be edited

### CODE REVIEW

File: content/refguide/published-rest-operation-parameter.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Specify the type of the parameter. Object or List parameters can only come from the body of the request.
+
+### 2.3 Parameter type
+
+Specify where the parameter comes from. Possible values are
+
+* **Query** When the request contains a query string such as `?name=John&age=42`, you can pass these to the microflow by adding query parameters. For more information, see [Published REST Query Parameters](published-rest-query-parameters).
+* **Path** The operation path can contain parameters as well. If you add a path parameter, make sure you add it to the operation path as well. For more information, see [Published REST Path Parameters](published-rest-path-parameters).
+* **Body** The microflow can have 0 or 1 body parameters. A body parameter is taken from the body of the request. If the body parameter has type Object or List, an import mapping is needed to convert the body content of the request into an object or a list. **GET**, **HEAD** and **OPTIONS** operations shouldn't have body parameters.
```
Comment (fazzdev) [inline content/refguide/published-rest-operation-parameter.md]: We need to explain here that storing the body as filedocument is possible too. Useful for storing images etc.

File: content/refguide/published-rest-operation-parameter.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Published REST Operation Parameter"
+parent: "published-rest-services"
+---
+
+{{% alert type="info" %}}
+
+This feature was introduced in version 7.17.0.
```
Comment (fazzdev) [inline content/refguide/published-rest-operation-parameter.md]: Check with Adam to see if we can do range here. Something like 'This feature was gradually introduced throughout version 7.13 until 7.17.'

File: content/refguide/published-rest-operation-parameter.md (hunk 1 above)
Comment (joostverhoog) [reply content/refguide/published-rest-operation-parameter.md]: Yes

File: content/refguide/published-rest-operation-parameter.md (hunk 2 above)
Comment (joostverhoog) [reply content/refguide/published-rest-operation-parameter.md]: Done

**Pull request merged**